lib/sequences/07_argonaut_search_sequence.rb

```diff
@@ -54,17 +54,30 @@ def validate_reply(klass, reply)
     assert_response_ok patient_read_response
     @patient = patient_read_response.resource
     assert !@patient.nil?, 'Expected valid DSTU2 Patient resource to be present'
-    @patient_hash = @patient.to_hash
     assert @patient.is_a?(FHIR::DSTU2::Patient), 'Expected resource to be valid DSTU2 Patient'
 
   end
 
+  test 'Patient search without authorization',
+          'http://www.fhir.org/guides/argonaut/r2/Conformance-server.html',
+          'A Patient search does not work without proper authorization' do
+
+    assert !@patient.nil?, 'Expected valid DSTU2 Patient resource to be present'
+    identifier = @patient.try(:identifier).try(:first).try(:value)
```

Comment:
Ditto on explicit about nil.  Same comment applies to all instances of this below.